public/ge0.html

```diff
@@ -76,29 +86,33 @@
 </head>
 
 <body>
-  <div id="header">
-    <span id="name">${name}</span>
-    <span>
-      <a id="open" href="om:/${path}" class="link">
-        Open in Organic Maps
-      </a>
-      <a href="https://omaps.app/get" target="_blank" class="link">
-        Install Organic Maps
-      </a>
-      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">
-        See on OpenStreetMap
-      </a>
-    </span>
+  <div id="header" class="box">
+    <div id="open" class="block">
+      <a href="om:/${path}" class="link">Show in Organic Maps</a>
+    </div>
+    <div id="osm" class="block">
+      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">See on OpenStreetMap</a>
+    </div>
+    <div class="block">
+      <a href="https://omaps.app/get" target="_blank" class="link">Get Organic Maps</a>
+    </div>
   </div>
   <div id="map" class="map"></div>
+
   <script type="text/javascript">
     const isiOS = navigator.platform.substr(0, 2) === 'iP' ||  // iPhone, iPad, iPod, including simulators.
       (navigator.userAgent.includes('Mac') && 'ontouchend' in document);  // iPad on iOS 13.
     const isAndroid = !isiOS && /(android)/i.test(navigator.userAgent);
     const isDesktop = !isiOS && !isAndroid;
+    coords = 'Lat: ${lat}, Lon: ${lon}';
 
     if (isDesktop)
+    {
       document.getElementById('open').remove();
+    } else {
+      document.getElementById('osm').remove();
```

Comment:
Why is it removed for mobiles?

Reply:
I believe, there is no any value to open OSM in browser on mobile when you have geo link. My initial intention was to minimize number of interface elements since now it look overloaded.

Reply:
The main goal of this stub is to allow users to see the POI and open it _anywhere_. Overloading is not important, opening and understanding what POI was shared is important.